test/integration/full-protocol/BaseAssetPairForkTest.sol

```diff
@@ -2,14 +2,15 @@
 pragma solidity 0.8.22;
 
 import "forge-std/Test.sol";
-import "./BaseProtocolForkTest.sol";
+
+import { ConfigHub } from "../../../src/ConfigHub.sol";
+import { BaseDeployer } from "../../../script/BaseDeployer.sol";
 import { ILoansNFT } from "../../../src/interfaces/ILoansNFT.sol";
 import { IRolls } from "../../../src/interfaces/IRolls.sol";
 import { ICollarTakerNFT } from "../../../src/interfaces/ICollarTakerNFT.sol";
 import { ICollarProviderNFT } from "../../../src/interfaces/ICollarProviderNFT.sol";
-import { BaseDeployer } from "../../../script/BaseDeployer.sol";
 
-abstract contract BaseLoansForkTest is BaseProtocolForkTest {
+abstract contract BaseAssetPairForkTest is Test {
```

Comment:
consolidated the two contracts, since for now we only have "asset pair tests" (loan tests)